app/ModelSerializers/Summit/Speakers/PresentationSpeakerSerializer.php

```diff
@@ -65,8 +65,10 @@ public function serialize($expand = null, array $fields = [], array $relations =
             $featured = $summit->getFeatureSpeaker($speaker);
             $values['featured']                = !is_null($featured);
             $values['order']                   = is_null($featured) ? 0 : $featured->getOrder();
-            if(in_array('presentations', $relations))
-                $values['presentations']           = $speaker->getPresentationIds($summit->getId(), $published);
+            if(in_array('presentations', $relations)) {
+                $values['presentations']    = $speaker->getPresentationIds($summit->getId(), $published);
+                $values['order_by_presentation'] = $speaker->getPresentationAssignmentsOrder();
```

Comment:
i dont see the reason to add this to the serializer, it just add overhead bc its a costly operation

Reply:
removed